tests/garage/experiment/test_experiment.py

```diff
@@ -82,3 +50,45 @@ def test_experiment_with_variant():
     assert len(folder_content_diff) == 1
     exp_folder_name = folder_content_diff.pop()
     assert exp_folder_name.startswith('test_prefix')
+
+
+# Pylint gets confused by @wrap_experiment adding an argument.
+# pylint: disable=no-value-for-parameter
```

Comment:
@ryanjulian This is actually a design point we need to discuss. `@wrap_experiment` injects an argument, which pylint has no clue how to handle.

Reply:
For now (in the examples), I've given the context a default value of None.

Reply:
so pylint will complain for every usage of wrap_experiment?

Reply:
Yes. It's quite annoying. I'm not quite sure what the best solution is.

Reply:
For now, I've made the `ctxt` parameter "optional", which is sufficient to make pylint happy. I appear to have deleted all of the example updates when I rebased this PR, so I'll push up changes to examples in a new PR.